app/frontend/package.json

```diff
@@ -35,16 +35,20 @@
     "bootstrap-scss": "^4.6.1",
     "core-js": "^3.23.3",
     "deep-freeze": "^0.0.1",
+    "export-from-json": "^1.7.2",
     "fast-json-patch": "^3.1.1",
     "font-awesome": "^4.7.0",
+    "formio-utils": "^0.2.4",
     "formiojs": "^4.14.6",
     "keycloak-js": "^15.0.2",
     "material-design-icons-iconfont": "^6.7.0",
     "moment": "^2.29.3",
     "nprogress": "^0.2.0",
+    "papaparse": "^5.4.0",
     "qrcode.vue": ">= 1.7.0 < 3.0.0",
     "uuid": "^8.3.2",
     "vue": "^2.7.2",
+    "vue-blob-json-csv": "^0.1.7",
```

Comment:
Do you guys have to use this package? Have you looked into json2csv?

Reply:
We it cause it allow us add a download button for JSON and CSV. but you package json2csv does not do what i want .in the frontend